client/cloudfront.tf

```diff
@@ -19,7 +19,7 @@ resource "random_pet" "lambda_bucket_name" {
 
 resource "aws_s3_bucket" "web_distribution" {
   bucket = random_pet.lambda_bucket_name.id
-  acl    = "private"
+  # acl    = "private"
```

Comment:
Did you speak to @prabhukiran9999 about his fix's for these issues? He mentioned he was deciding on which route to take this morning I think.